plugin/src/main/plugin-metadata/plugin-security.policy

```diff
@@ -24,4 +24,9 @@ grant {
 
     // Circuit Breaker
     permission java.lang.RuntimePermission "getFileSystemAttributes";
+    
+    // AWS credentials needed for clients
+    permission java.io.FilePermission "<<ALL FILES>>", "read";
+    
+    permission java.security.AllPermission;
```

Comment:
This essentially bypasses all security checks in the Java Security Manager and gives the code unrestricted access to the system. Why do we want that?

Reply:
![one ring to rule them all](https://upload.wikimedia.org/wikipedia/commons/6/6e/One_Ring.png?20240620142610)

But seriously, I think that was leftover from some debugging and I need to remove it.